pkg/steps/multi_stage.go

```diff
@@ -551,11 +555,30 @@ func (s *multiStageTestStep) runPods(ctx context.Context, pods []coreapi.Pod, sh
 }
 
 func (s *multiStageTestStep) runPod(ctx context.Context, pod *coreapi.Pod, notifier *TestCaseNotifier) error {
+	// work around activeDeadlineSeconds vs. container status: https://github.com/kubernetes/kubernetes/issues/88611
+	activeDeadlineSeconds := pod.Spec.ActiveDeadlineSeconds
+	pod.Spec.ActiveDeadlineSeconds = nil
 	if _, err := createOrRestartPod(s.podClient.Pods(s.jobSpec.Namespace()), pod); err != nil {
 		return fmt.Errorf("failed to create or restart %q pod: %w", pod.Name, err)
 	}
+	podComplete := make(chan struct{}, 1)
```

Comment:
I think I'd prefer to see this as a context that gets cancelled

Reply:
> I think I'd prefer to see this as a context that gets cancelled

Done with cb9b7921d -> b97fd1cee.